homeassistant/components/spc/manifest.json

```diff
@@ -2,8 +2,9 @@
   "domain": "spc",
   "name": "Vanderbilt SPC",
   "codeowners": [],
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/spc",
   "iot_class": "local_push",
   "loggers": ["pyspcwebgw"],
-  "requirements": ["pyspcwebgw==0.4.0"]
+  "requirements": ["pyspcwebgw==0.6.0"]
```

Comment:
There should be a link in the PR description to the release or something for this dependency upgrade

Reply:
covered at https://github.com/home-assistant/core/pull/87915#issuecomment-1426903807